scripts/globals/mobskills/armor_buster.lua

```diff
@@ -10,20 +10,16 @@ require("scripts/globals/status")
 local mobskillObject = {}
 
 mobskillObject.onMobSkillCheck = function(target, mob, skill)
-    if mob:getLocalVar("citadelBuster") == 0 then
```

Comment:
Note: all of the following code sections were dead code.
No where was the citadelBuster localVar being set - so it always evaluated to true.
I'm guessing its from an older version of the mob.
```
 if mob:getLocalVar("citadelBuster") == 0 then
        return 0
end
```